src/runtime/js.zig

```diff
@@ -1393,6 +1404,36 @@ pub fn Env(comptime State: type, comptime WebApis: type) type {
                         },
                         else => {},
                     },
+                    .array => |arr| {
+                        // Retrieve fixed-size array as slice then probe
+                        const slice_type = []arr.child;
+                        switch (try self.probeJsValueToZig(named_function, slice_type, js_value)) {
+                            .value => |slice_value| {
+                                if (slice_value.len == arr.len) {
+                                    return .{ .ok = {} };
```

Comment:
Should consider returning a `.{.value =  ...}` here. If probe returns a value, that means there was a cost to creating it that it doesn't want you to have to repeat. In this case, the probe allocated + populated the slice). If you turn that `.value` into an `.ok`, it'll have to do that same allocation + copy again.